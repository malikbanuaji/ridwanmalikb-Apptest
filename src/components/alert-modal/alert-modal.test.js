import * as React from 'react';

import {render} from '@testing-library/react-native';
import AlertModal from './alert-modal';

jest.mock('react-native-modal');

test('should render properly', () => {
  const Content = 'Content';
  const component = render(
    <AlertModal isVisible={false}>
      <Content />
    </AlertModal>,
  );
  expect(component.toJSON()).toMatchSnapshot();
});
