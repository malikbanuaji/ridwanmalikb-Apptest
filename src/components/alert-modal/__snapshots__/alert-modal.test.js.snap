// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
<Modal
  animationIn="fadeIn"
  animationInTiming={200}
  animationOut="fadeOut"
  backdropColor="#000000"
  backdropOpacity={0.8}
  isVisible={false}
  style={
    Object {
      "margin": 0,
    }
  }
  useNativeDriver={true}
  useNativeDriverForBackdrop={true}
>
  <Content />
</Modal>
`;
